pkg/util/stringset.go

```diff
@@ -6,42 +6,56 @@ import (
 
 type StringSet struct {
 	syncMutex *sync.Mutex
-	Set       map[string]bool
+	set       map[string]bool
 }
 
 func NewStringSet() *StringSet {
 	return &StringSet{
 		syncMutex: &sync.Mutex{},
-		Set:       make(map[string]bool)}
+		set:       make(map[string]bool)}
 }
 
 func (set *StringSet) Add(s string) bool {
 	set.syncMutex.Lock()
 	defer set.syncMutex.Unlock()
 
-	_, found := set.Set[s]
-	set.Set[s] = true
+	_, found := set.set[s]
+	set.set[s] = true
 	return !found
 }
 
 func (set *StringSet) Contains(s string) bool {
 	set.syncMutex.Lock()
 	defer set.syncMutex.Unlock()
 
-	_, found := set.Set[s]
+	_, found := set.set[s]
 	return found
 }
 
 func (set *StringSet) Size() int {
 	set.syncMutex.Lock()
 	defer set.syncMutex.Unlock()
 
-	return len(set.Set)
+	return len(set.set)
 }
 
 func (set *StringSet) Delete(s string) {
 	set.syncMutex.Lock()
 	defer set.syncMutex.Unlock()
 
-	delete(set.Set, s)
+	delete(set.set, s)
+}
+
+func (set *StringSet) Elements() []string {
+	set.syncMutex.Lock()
+	defer set.syncMutex.Unlock()
+
+	elements := make([]string, len(set.set))
+	i := 0
+	for v, _ := range set.set {
```

Comment:
This trips linting, it should be

    for v := range set.set {

Reply:
Done